Objective-C/Internal/CBLVersion.m

```diff
@@ -44,13 +45,28 @@ + (NSString*) userAgent {
         NSString* platform = strcmp(CBL_PRODUCT_NAME, "CouchbaseLiteSwift") == (0) ?
         @"Swift" : @"ObjC";
         
-        NSString* commit = strlen(GitCommit) > (0) ?
-        [NSString stringWithFormat: @"Commit/%.8s%s", GitCommit, GitDirty] : @"NA";
+        
+        NSString* ceCommit = [NSString stringWithFormat:@"%.8s", GitCommit];
+        NSString* eeCommit = [NSString stringWithFormat:@"%.8s", GitCommitEE];
+
+        NSMutableString* commit = [NSMutableString stringWithString:ceCommit];
+
+        // Add +EE hash only if eeCommit is not empty
+        if (GitCommitEE && GitCommitEE[0]) {
+            [commit appendFormat:@"+%@", eeCommit];
+             type = @"-EE";
+        } else {
+             type = @"-CE";
+        }
+        
+        if ((GitDirty && GitDirty[0]) || (GitDirtyEE && GitDirtyEE[0])) {
+            [commit appendString:@"+CHANGES"];
+        }
         
         C4StringResult liteCoreVers = c4_getVersion();
         
-        sUserAgent = [NSString stringWithFormat: @"CouchbaseLite/%s (%@; %@) Build/%d %@ LiteCore/%.*s",
-                      CBL_VERSION_STRING, platform, system, CBL_BUILD_NUMBER, commit,
+        sUserAgent = [NSString stringWithFormat: @"CouchbaseLite/%s%@ (%@; %@) Build/%d Commit/%@ LiteCore/%.*s",
```

Comment:
Can you confirm that the format to provide the enterprise/community edition (e.g. -EE/-CC or EE commit) is inline with the format that the other platforms (.NET) is using?

For example, I was looking at https://github.com/couchbase/couchbase-lite-net/blob/master/src/Couchbase.Lite.Shared/Sync/HTTPLogic.cs#L196-L219 but it's not clear if -EE or -CE suffix is appended.